### ISSUE TITLE
How to represent the [mask] token?

### ISSUE THREAD

**Issue by shinthor** [TencentAILabHealthcare/scBERT#25]
Hello, I am wondering how did you mask the expression levels for pre-training? I'm assuming a vector of 200 zeros would work, but that it would be indistinguishable from actual zeros in the data, so maybe something like a vector of 200 -1's would be better/more explicit? What worked for you?

**Comment by TencentAILabHealthcare** [comment]
We have indicators to distinguish whether they have been randomly masked. The pre-training codes have been released, please refer to    https://github.com/TencentAILabHealthcare/scBERT/blob/master/pretrain.py   for details.
